src/System.Resources.ResourceManager/ref/project.json

```diff
@@ -1,14 +1,13 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Reflection": "4.0.0",
-    "System.Globalization": "4.0.0"
+    "System.Collections.NonGeneric": "4.3.0-beta-24604-02",
```

Comment:
All these new dependencies are likely to introduce cycles in the package graph, some may be cycles in the assembly graph too.